index.d.ts

```diff
@@ -1574,6 +1582,34 @@ declare namespace Eris {
     filename?: string;
     id: string | number;
   }
+  interface Poll {
+    allow_multiselect: boolean;
+    answers: PollAnswer[];
+    expiry: string | null;
+    layout_type: number;
+    question: PollMedia;
```

Comment:
This should only read from `text`, or rather `emoji` should be removed. Also `text` is required

Reply:
Fixed. `emoji` is used in poll answers so I've kept that but removed the Partial type 👍🏻